src/Switch-mbTechnologies_HVM/main.py

```diff
@@ -192,63 +210,75 @@ def get_identification(self):
         return answer
     
     def get_operation_complete(self):
-        """
-        Waits until all pending operations are complete then returns 1
-        
+        """Waits until all pending operations are complete then returns 1
+
+        Returns:
+            bool: True if operation complete finished. Otherwise it will timeout.
         """
         self.port.write("*OPC?")
-        if self.port.read() == "1":
-            return True
-        else:
-            return False
+        return self.port.read() == "1"
 
     def reset(self) -> None:
-        """
-        Resets the instrument and opens all relays. 
+        """Resets the instrument and opens all relays.
         """
         self.port.write("*RST")
+        if not self.port_string.startswith("GPIB"):
+            self.port.read()
```

Comment:
I've seen this line several times. Why is GPIB excluded? Is this something specific to this instrument? Or is that common knowledge for GPIB in general?

Reply:
It is related to the software protocol of the instrument. In case of GPIB no answer is sent for function hat set a property, whereas for all other interfaces a message is always returned.

Reply:
I added a further note to the code.